hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -41,6 +41,16 @@
             newUpdater(NearCacheStatsImpl.class, "evictions");
     private static final AtomicLongFieldUpdater<NearCacheStatsImpl> EXPIRATIONS =
             newUpdater(NearCacheStatsImpl.class, "expirations");
+    private static final AtomicLongFieldUpdater<NearCacheStatsImpl> PERSISTENCE_COUNT =
```

Comment:
Are all fields that have FieldUpdater instances accessed concurrently and are the FieldUpdater instances needed? If they are only accessed by one thread at a time, they can just be volatile. E.g. I believe that `PERSISTENCE_COUNT` is accessed by only one thread. But I could be wrong.